ios/App/Shared/player/AudioPlayer.swift

```diff
@@ -57,9 +62,9 @@ class AudioPlayer: NSObject {
         self.playWhenReady = playWhenReady
         self.initialPlaybackRate = playbackRate
         self.audioPlayer = AVQueuePlayer()
-        self.audioPlayer.automaticallyWaitsToMinimizeStalling = false
+        self.audioPlayer.automaticallyWaitsToMinimizeStalling = true
```

Comment:
Are you sure this is required? If I remember correctly this flag was causing issues with playing HLS transcodes.
I could be wrong on that so I'm wondering what caused you to enable this.

Reply:
I'm new to `AVPlayer` and streaming audio. Trying to read up on how it all works.
I'll try to make sure to test with HLS streams and it would be great if you can do that too.

But when I read the docs I came away wondering why you decided to set it to `false`. Could the HLS issues be connected to other decisions on how to handle the playing?

The [docs](https://developer.apple.com/documentation/avfoundation/avplayer/1643482-automaticallywaitstominimizestal) say:
```
When playing media delivered over HTTP, this property is used to determine 
if the player should automatically delay playback in order to minimize stalling.
```
So when set to `true` the player tries to handle stalls itself.
If we set it to `false` we should handle it ourselves:
```
You will need to set this property to false when you require precise control over playback start times, 
such as if you're are synchronizing multiple player instances using the setRate(_:time:atHostTime:) method.
```
And we're not handling it.

It's good to note that:
```
If the value of this property is false, playback will start immediately when requested 
as long as the playback buffer is not empty.
```
So set to `false` the playback will start faster but, with the current code, we could end up needing to buffer and the player will be paused. And this is why my code had issues while streaming.

If you try my PR and compare it to the old code while seeking you will feel how much better the player behaves.

Reply:
I will do some testing today. HLS may be a problem because we are starting playback before the full audiobook is transcoded. I never figured out exactly how AVPlayer is handling this because their documentation is so slim, but what happens is it will request segments that haven't been created on the server yet.

On Android we can control how far out it will buffer and same with the website. AVPlayer tries to load way too much and when the server responds with an error that it hasn't been created yet, then it stalls the playback.

It has been a while now since I dove into this so I'll look into it again today now that I have a little more experience with iOS.